### DISCUSSION TITLE
PieChart: Image alignment changes with legend size

### DISCUSSION THREAD

**Discussion by maemigh** [grafana/grafana#38240] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
When stacking multiple PieChart panels and legends are different, the pie chart image alignment changes and appears off:

<img width="384" alt="Screen Shot 2021-03-15 at 22 37 34" src="https://user-images.githubusercontent.com/4478053/111162683-ba9d3c00-85df-11eb-8b33-146ccd591803.png">

**Why is this needed**:
Cleaner look

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
